recipes/njoy2016/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set org = "njoy" %}
+{% set name = "NJOY2016" %}
+{% set version = "2016.66" %}
+{% set build = 0 %}
```

Comment:
Only the version should be a jinja variable, to make it easy to update, the rest should not change and it is OK to "hardcode" them below. In fact it makes the recipe more readable and "clickable" with working URLs.